### PR TITLE
PostgreSQL Container for Docker Cluster

### PR DESCRIPTION
**Pull request by bodo-hugo-barwich**
This introduces support for _PostgreSQL_ for the **Docker Cluster**.
As discussed in  #26 

It's important to mention that the `DBD::Pg` _Perl_ Module required _PostgreSQL_ development libraries on the Web Service container which are only available for the **_PostgresSQL_ Version 11**
The databases of Version 10 and 11 are incompatible so the version of the _PostgreSQL_ container is `11.13`
According to the **End of Life Policy** at _Postgres_ this seems to be a reasonable versioning:
[_Postgres_ Versioning Chart](https://www.postgresql.org/support/versioning/)

### CODE REVIEW

**Comment by bodo-hugo-barwich** [comment]
The access from the Web Service container is documented as correct
according to the `Mojolicious` documentation:
https://docs.mojolicious.org/Mojo/Pg#SYNOPSIS
```
per1_web@b2046f460754:~$ perl -I ~/perl5/lib/perl5/ perldoc-browser.pl index all
pg 0 dmp:
bless({
  dsn => "dbi:Pg:dbname=perldoc;host=db",
  options => {
    AutoCommit          => 1,
    AutoInactiveDestroy => 1,
    PrintError          => 0,
    PrintWarn           => 0,
    RaiseError          => 1,
  },
  password => "secret",
  username => "perldoc",
}, "Mojo::Pg")
pg version: 'PostgreSQL 11.13 on x86_64-pc-linux-musl, compiled by gcc (Alpine 10.3.1_git20210424) 10.3.1 20210424, 64-bit'
```
also in the _PostgreSQL_ container the entries can be found correctly:
```
$ docker-compose exec db /bin/sh
/ $ psql -h db -U perldoc perldoc
Password for user perldoc: 
psql (11.13)
Type "help" for help.

perldoc=# \dt
             List of relations
 Schema |      Name       | Type  |  Owner  
--------+-----------------+-------+---------
 public | faqs            | table | perldoc
 public | functions       | table | perldoc
 public | mojo_migrations | table | perldoc
 public | perldeltas      | table | perldoc
 public | pods            | table | perldoc
 public | variables       | table | perldoc
(6 rows)

perldoc=# select count(*) from pods;
 count 
-------
  1456
(1 row)
```

File: .gitignore [hunk 1]
```diff
@@ -17,5 +17,10 @@ hypnotoad.pid
 perl5
 perl5/*
 
+#Database Container Storage
+data
+data/*
+~data/.keep
```
Comment (Grinnz) [inline .gitignore:23]: Should there be a leading `~` here? I'm not aware of tilde meaning anything to gitignore

Comment (bodo-hugo-barwich) [reply .gitignore]: Yes, you are right. 
This was rather meant to keep the file and the directory so it will be distributed to new installations.

**Comment by bodo-hugo-barwich** [comment]
to resolve the permission conflict at image build time I added the `.dockerignore` 
as documented at
[Conflict about persistent Database Storage Location](https://github.com/Grinnz/perldoc-browser/issues/26#issuecomment-903264572)

**Pull request merged**